app/src/androidTest/java/org/mozilla/fenix/syncintegration/SyncIntegrationTest.kt

```diff
@@ -207,8 +209,7 @@ class SyncIntegrationTest {
     }
 
     fun historyAfterSyncIsShown() {
-        val historyEntry = mDevice.findObject(By.text("http://www.example.com/"))
-        historyEntry.isEnabled()
+        mDevice.waitNotNull(Until.findObjects(By.text("http://www.example.com/")), TestAssetHelper.waitingTime)
```

Comment:
I have to investigate that, on iOS using a static asset did not work because the history was not properly recorded on the iOS device. Also, in this case we are checking on Fenix that the history created on Desktop is shown, so it is kind of static from Fenix side.

I will investigate if creating the history using a static asset on Fenix works, but if possible let's keep that for the follow up PR too